rate: 50.0
arm_joints: ['shoulder_pan', 'shoulder_pitch','elbow_flex','wrist_roll']

